integration_tests/loop_10.py

```diff
@@ -49,9 +49,68 @@ def nested_loop_for_for():
         return
     assert False
 
+def nested_loop_for_while():
+    i: i32
+    j: i32 = 10
+    for i in range(2):
+        print("outer: " + str(i))
+        while j < 20:
+            print("  inner: " + str(j))
+            break
+    else:
+        print("no break in outer loop")
+        return
+    assert False
+
+def nested_loop_while_for():
+    i: i32 = 0
+    j: i32
+    while i < 2:
+        print("outer: " + str(i))
+        i += 1
+        for j in range(10, 20):
+            print("  inner: " + str(j))
+            break
+    else:
+        print("no break in outer loop")
+        return
+    assert False
+
+def nested_loop_while_while():
+    i: i32 = 0
+    j: i32 = 10
+    while i < 2:
+        print("outer: " + str(i))
+        i += 1
+        while j < 20:
+            print("  inner: " + str(j))
+            break
+    else:
+        print("no break in outer loop")
+        return
+    assert False
+
+def nested_loop_else():
+    i: i32
+    j: i32
+    for i in range(2):
+        print("outer: " + str(i))
+        for j in range(10, 12):
+            print("  inner: " + str(j))
+        else:
+            print("no break in inner loop")
```

Comment:
You can also add assert statement on variables. For example increment a variable in this else portion and later have an assert on the value of the variable.

Similarly we can do for the above tests.